api/src/pcapi/routes/backoffice/offerers.py

```diff
@@ -0,0 +1,188 @@
+import sqlalchemy as sa
+
+from pcapi.core.bookings import models as bookings_models
+from pcapi.core.educational import models as educational_models
+from pcapi.core.finance import models as finance_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import models as offers_models
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.permissions import utils as perm_utils
+from pcapi.models import db
+from pcapi.models.api_errors import ApiErrors
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+from . import serialization
+from . import utils
+
+
+@blueprint.backoffice_blueprint.route("offerers/<int:offerer_id>", methods=["GET"])
+@spectree_serialize(
+    response_model=serialization.Response,
+    on_success_status=200,
+    api=blueprint.api,
+)
+@perm_utils.permission_required(perm_models.Permissions.READ_OFFERER)
+def get_offerer(offerer_id: int) -> serialization.Response:
+    ko_bank_informations_query = (
+        sa.select(sa.func.count(offerers_models.Venue.id))
+        .select_from(offerers_models.Venue)
+        .outerjoin(finance_models.BankInformation)
+        .filter(
+            offerers_models.Venue.managingOffererId == offerers_models.Offerer.id,
+            sa.or_(
```

Comment:
peut-être qu'on peut simplifier ce `or_ `en :
sa.not(status == ACCEPTED) ?

Reply:
c'est ce que j'avais essayé en premier, mais curieusement, ça me fonctionne pas.
apparemment il faut vraiment spécifier explicitement qu'on veut aussi le cas ou le status est `None`.